README.md

```diff
@@ -96,9 +106,9 @@ Instantly get access to our documents and meeting invites at http://kubestellar.
 <br>
 
 <td>
-    <a href="https://landscape.cncf.io">
-        <img src="/docs/overrides/images/cncf-color.png" width="300px;" alt="Cloud Native Computing Foundation Logo"/>
-    </a>
+    <a href="https://landscape.cncf.io">
+        <img src="/docs/overrides/images/cncf-color.png" width="300px;" alt="Cloud Native Computing Foundation Logo"/>
+    </a>
```

Comment:
GitHub's display here does not make it obvious, but the change is replacing some space characters (U+0020) with non-breaking space characters (U+00A0). Please do not do this.